MetaMask/metamask-extension
# Replaces SEVERITIES const with Severity enum in BannerAlert

## Explanation
- This PR fixes Part of #19528 and #18714 
- This Replaces all instances of `SEVERITIES` const with `Severity` enum in `ui/components/component-library/banner-alert` folder

## Screenshots/Screencaps



### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/c8654c79-40a3-42d3-a17c-19deb05d3981



### After

https://github.com/MetaMask/metamask-extension/assets/8112138/20d21481-600a-4902-b240-9a8dc080aa3f

## Manual Testing Steps

- Pull this branch
- Run storybook `yarn storybook`
- Search `BannerAlert` in storybook
- Check docs and stories

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
